src/site/stages/build/drupal/metalsmith-drupal.js

```diff
@@ -131,9 +130,9 @@ function paginatePages(page, files, field, layout, ariaLabel, perPage) {
   }
 }
 
-// Return page object with path, breadcrump and title set.
-function createEntityUrl(page, drupalPagePath, title) {
-  const pathSuffix = title.replace(/\s+/g, '-').toLowerCase();
+// Return page object with path, breadcrumb and title set.
+function createEntityUrl(page, drupalPagePath, title, pathSuffix) {
+  pathSuffix = pathSuffix || title.replace(/\s+/g, '-').toLowerCase();
```

Comment:
needed a way to pass in "stories" as the path and "Community stories" as the title!